contracts/interfaces/registries/ILicenseRegistryV2.sol

```diff
@@ -0,0 +1,81 @@
+// SPDX-License-Identifier: BUSL-1.1
+pragma solidity 0.8.23;
+
+import { Licensing } from "../../lib/Licensing.sol";
+
+/// @title ILicenseRegistry
+interface ILicenseRegistryV2 {
+    event LicenseTemplateRegistered(address indexed licenseTemplate);
+    event RoyaltyPolicyRegistered(address indexed royaltyPolicy);
+    event CurrencyTokenRegistered(address indexed token);
+    event MintingLicenseSpecSet(
```

Comment:
have we decided to call it licenseSpec?  I thought we have a different name. please check slack messages.